middleman-core/lib/middleman-core/server_listener.rb

```diff
@@ -0,0 +1,72 @@
+require 'socket'
+require 'resolv'
+
+module Middleman
+  class ServerListener
+    private
+
+    attr_reader :https
```

Comment:
No need to have this, just use `@https` in the code. Also you could have avoided the `private`/`public` dance by calling `private :https`.